Antidote/ChatViewController.m

```diff
@@ -373,4 +384,21 @@ - (void)updateLastReadDate
     [[AppContext sharedContext].profileManager.toxManager.objects changeChat:self.chat lastReadDateInterval:interval];
 }
 
+- (void)startCallButtonPressed
+{
+    OCTSubmanagerCalls *manager = [AppContext sharedContext].profileManager.toxManager.calls;
+
+    OCTCall *call = [manager callToChat:self.chat enableAudio:YES enableVideo:NO error:nil];
```

Comment:
Well, similarly to regular phone, Skype, Viber, whatever, user will always call with audio by default. If she likes she can disable audio right after start of dialing.